test/suite/test_hs07.py

```diff
@@ -81,14 +81,16 @@ def test_hs(self):
         ds2 = SimpleDataSet(self, uri2, 0, key_format=self.key_format, value_format="S")
         ds2.populate()
 
-        # Pin oldest and stable to timestamp 1.
-        self.conn.set_timestamp('oldest_timestamp=' + self.timestamp_str(1) +
-            ',stable_timestamp=' + self.timestamp_str(1))
 
         bigvalue = "aaaaa" * 100
         bigvalue2 = "ddddd" * 100
+        # Commit at timestamp 1.
```

Comment:
Nitpick: line break between `bigvalue2 = ...` and `# Commit at ...` would keep this closer to the original change.

Reply:
[2c84acb](https://github.com/wiredtiger/wiredtiger/pull/10088/commits/2c84acb9ab2967d5f81bc1d18107430f965ce2fb)